### ISSUE TITLE
How to train MPRNet on GoPro dataset by using proposed loss function?

### ISSUE THREAD

**Issue by jacky1107** [swz30/MPRNet#5]
Hi, your MPRNet is excellent. I also work on image deblurring tasks, but have been struggling to design a good loss function. I'm wondering that could you write some example of train.py, and I'm also curious about how do implement proposed loss function.
Thanks.

**Comment by swz30** [comment]
Hi @jacky1107,

The implementation of the loss function is provided at https://github.com/swz30/MPRNet/blob/main/Deblurring/losses.py

Thanks
